### PR TITLE
My additions

### PR DESCRIPTION
**Pull request by Jason-Bengtson**
excellent

### CODE REVIEW

**Review by Jason-Bengtson — COMMENTED** [review]
well done me

File: ARTICLES.md [hunk 1]
```diff
@@ -16,5 +16,12 @@ Place your article suggestions at the end of this list.  When you create the PR
...
  - Quick list of good tips for developers in 2020 
+
+
+
+[German taxpayers faced with €800k Windows 7 support bill due to Deutschland dithering](https://www.theregister.co.uk/2020/01/22/germany_windows_7_support_bill/)
+- An article highlighting the potential costs of maintaining older systems. Thousands of German government computers are still using the windows 7 operating systems, which windows is no longer supporting with updates (including for security).
+=======
 [More Valuable than Oil, Data Reigns in Today’s Data Economy](https://www.northridgegroup.com/blog/more-valuable-than-oil-data-reigns-in-todays-data-economy/)
 - How important it is becoming to be able refine the raw data comapanies recieve to create valuable interpretable information. There is no shortage of data and it is potentially the most important part of the business structure in todays world.
+
```
Comment (Jason-Bengtson) [inline ARTICLES.md]: looks good

File: favorite/MOVIES.md [hunk 2]
```diff
@@ -1,13 +1,8 @@
...
-Howl's Moving Castle
->>>>>>> 656db1ed260175dd1a5a9dce353de1cc9e9d8b75
+
++ The Shawshank Redemption
++ Midsommar
++ Howl's Moving Castle
++ The Big Lebowski / Ace Ventura When Nature Calls
+
+
+
```
Comment (Jason-Bengtson) [inline favorite/MOVIES.md]: check

File: favorite/NETFLIX.md [hunk 3]
```diff
@@ -2,4 +2,6 @@
 + Dark
 + Peaky Blinders
 + Mind Hunter
-+ Breaking Bad
\ No newline at end of file
++ Breaking Bad
++ Human Centipede
++ The Witcher
```
Comment (Jason-Bengtson) [inline favorite/NETFLIX.md]: check

File: favorite/MOVIES.md [hunk 4]
```diff
@@ -1,13 +1,8 @@
...
-Howl's Moving Castle
->>>>>>> 67bff4eae496b3d67b6a1e56697832650129c3de
-=======
-Midsommar
-Howl's Moving Castle
->>>>>>> 656db1ed260175dd1a5a9dce353de1cc9e9d8b75
+
++ The Shawshank Redemption
++ Midsommar
++ Howl's Moving Castle
```
Comment (Jason-Bengtson) [inline favorite/MOVIES.md]: check

File: favorite/BOOKS.md [hunk 5]
```diff
@@ -1 +1,2 @@
-- Mistborn - Brandon Sanderson
\ No newline at end of file
+- Mistborn - Brandon Sanderson
+- The Call of Cthulhu - H.P Lovecraft - http://www.hplovecraft.com/writings/texts/fiction/cc.aspx
```
Comment (Jason-Bengtson) [inline favorite/BOOKS.md]: check

**Review by sleddog — APPROVED** [review]
If this includes all the changes from the previous PRs, feel free to merge this one and close the other PRs.

**Pull request merged**